src/components/DpButton/DpButton.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+
+import DpButton from './DpButton.vue'
+
+const meta: Meta<typeof DpButton> = {
+    component: DpButton,
+    title: "Components/Button"
+}
+
+export default meta
+type Story = StoryObj<typeof DpButton>
+
+export const Default: Story = {
+    args: {
+        busy: false,
+        color: 'secondary',
+        hideText: false,
+        icon: 'delete',
+        iconSize: 'medium',
+        rounded: true,
+        text: 'Delete',
+        type: 'submit',
+        variant: 'outline'
+    }
+}
```

Comment:
shouldn't be the Default quite "default". Here You set several props to not so default values ;-)

Reply:
https://github.com/demos-europe/demosplan-ui/pull/397/commits/3b0f3d620777ef82c0b2777830607ffa6910e3d5

Reply:
but why do You define them at all? can't You just omit the props (they are all not required)

Reply:
sorry, yes we can
https://github.com/demos-europe/demosplan-ui/pull/397/commits/a0fcebfe307df57f1b0ad9c37a874cbb69d2d5d1

thanks